testing/bdd/features/timing/resolved_times.feature

```diff
@@ -41,10 +47,10 @@ Feature: Resolved times computation in sequence
     And doc3 has resolved end time <r_end_doc3>
 
     Examples:
-    | doc3_avail_time | doc3_begin  | doc3_end    | doc3_dur | r_begin_doc1 | r_end_doc1 | r_begin_doc2 | r_end_doc2 | r_begin_doc3 | r_end_doc3 |
-    | 00:00:20.0      | 00:00:50.0  | 00:01:00.0  |          | 00:00:10.0   | 00:00:20.0 | 00:00:30.0   | 00:00:40.0 | 00:00:50.0   | 00:01:00.0 |
-    | 00:00:03.0      | 00:00:50.0  | 00:01:00.0  |          | 00:00:10.0   | 00:00:20.0 | 00:00:30.0   | 00:00:40.0 | 00:00:50.0   | 00:01:00.0 |
-    | 00:00:20.0      | 00:00:35.0  | 00:01:00.0  |          | 00:00:10.0   | 00:00:20.0 | 00:00:30.0   | 00:00:35.0 | 00:00:35.0   | 00:01:00.0 |
+    | doc1_no_body | doc2_no_body | doc3_no_body | doc3_avail_time | doc3_begin | doc3_end   | doc3_dur | r_begin_doc1 | r_end_doc1 | r_begin_doc2 | r_end_doc2 | r_begin_doc3 | r_end_doc3 |  
```

Comment:
Would be better to move `doc1_no_body`, `doc2_no_body` and `doc3_no_body` to the scenario header at the top since they're all the same values in all the examples. Then the other lines would be both shorter and unchanged - more elegant.